commands/watch.ts

```diff
@@ -181,11 +206,31 @@ exports.builder = yargs => {
     default: false,
   });
 
-  addConfigOptions(yargs);
-  addAccountOptions(yargs);
-  addCmsPublishModeOptions(yargs, { write: true });
-  addUseEnvironmentOptions(yargs);
-  addGlobalOptions(yargs);
-
-  return yargs;
+  return yargs as Argv<WatchCommandArgs>;
+}
+
+const builder = makeYargsBuilder<WatchCommandArgs>(
+  watchBuilder,
+  command,
+  describe,
+  {
+    useConfigOptions: true,
+    useAccountOptions: true,
+    useGlobalOptions: true,
+    useEnvironmentOptions: true,
+    useCmsPublishModeOptions: { write: true },
+  }
+);
+
+const watchCommand: YargsCommandModuleBucket = {
```

Comment:
Looks like the handler is typed weird here because this is using the YargsCommandModuleBucket type instead of the YargsCommandModule type